WordPress/Classes/ViewRelated/Gutenberg/Processors/GutenbergImgUploadProcessor.swift

```diff
@@ -14,79 +13,58 @@ class GutenbergImgUploadProcessor: Processor {
         self.remoteURLString = remoteURLString
     }
 
-    lazy var imgPostMediaUploadProcessor = HTMLProcessor(for: "img", replacer: { (img) in
-        guard let imgClassAttributeValue = img.attributes["class"]?.value,
-            case let .string(imgClass) = imgClassAttributeValue else {
-                return nil
-        }
-
-        let classAttributes = imgClass.components(separatedBy: " ")
-
-        guard let imageIDAttribute = classAttributes.filter({ (value) -> Bool in
-            value.hasPrefix(GutenbergImgUploadProcessor.imgClassIDPrefixAttribute)
-        }).first else {
-            return nil
-        }
-
-        let imageIDString = String(imageIDAttribute.dropFirst(GutenbergImgUploadProcessor.imgClassIDPrefixAttribute.count))
-        let imgUploadID = Int32(imageIDString)
+    func processImgTags(_ block: GutenbergParsedBlock) {
+        let imgTags = try? block.elements.select("img")
+        imgTags?.forEach { img in
```

Comment:
Same comment about adding an `if let` or `guarding` to remove the optional.

Reply:
I applied this suggestion in https://github.com/wordpress-mobile/WordPress-iOS/pull/22896/commits/d60c1d19412f8df4723a174ab3982ea0af4fe6f1.